src/webui/www/public/scripts/mocha-init.js

```diff
@@ -451,6 +451,37 @@ initializeWindows = function() {
         }
     };
 
+    copyNameFN = function() {
+        var h = torrentsTable.selectedRowsIds();
```

Comment:
You're adding new code and no one currently is maintaining that file, so unless you plan to clean/rename it up later, it's easier to give it a better name now.